protocol/0080-SPOT-product_builtin_spot.md

```diff
@@ -125,6 +124,16 @@ Price-monitoring auctions are still required and should be implemented following
 1. It is not possible to change the `quote_asset` via governance (<a name="0080-SPOT-002" href="#0080-SPOT-002">0080-SPOT-002</a>)
 1. It is not possible to change the `base_asset` via governance (<a name="0080-SPOT-003" href="#0080-SPOT-003">0080-SPOT-003</a>)
 1. A `Spot` market can be closed through governance (<a name="0080-SPOT-004" href="#0080-SPOT-004">0080-SPOT-004</a>)
-1. Parties are unable to place orders they do not have the necessary funds for (<a name="0080-SPOT-005" href="#0080-SPOT-005">0080-SPOT-005</a>)
 1. Parties are unable to submit liquidity commitments they do not have the necessary funds for (<a name="0080-SPOT-006" href="#0080-SPOT-006">0080-SPOT-006</a>)
-1. Market liquidity fees are calculated correctly (<a name="0080-SPOT-007" href="#0080-SPOT-007">0080-SPOT-007</a>)
+1. If a "sell" order does not trade immediately (or only trades in part), an amount of the base_asset should be transferred to a holding_account to cover the remaining size of the order for the base_asset.(<a name="0080-SPOT-009" href="#0080-SPOT-009">0080-SPOT-009</a>)
+1. Since "sell" orders require a party to hold a sufficient amount of the base_asset to fulfill the order size, it is unnecessary to consider individual cases where fees may be incurred. If a "sell" order incurs fees through trading (i.e., by being the aggressor or trading in an auction), the required amount of the quote_asset to cover the fees will be deducted from the party's total quote_asset amount resulting from the sale of the base_asset.(<a name="0080-SPOT-010" href="#0080-SPOT-010">0080-SPOT-010</a>)
+1. As "buy" orders require a party to hold a sufficient amount of the `quote_asset` to cover possible fees, the individual cases where fees can be incurred must be considered.(<a name="0080-SPOT-011" href="#0080-SPOT-011">0080-SPOT-011</a>)
+1. For a "buy" order to be considered valid during continuous trading, the party must have a sufficient amount of the `quote_asset` in the `general_account` to cover the value of the trade as well as any possible fees incurred as a result of the order trading immediately (the aggressor).(<a name="0080-SPOT-012" href="#0080-SPOT-012">0080-SPOT-012</a>)
+1.If a "buy" order does not trade immediately (or only trades in part), only the necessary amount of the quote_asset to cover the remaining size of the order should be transferred to a holding_account for the quote_asset.(<a name="0080-SPOT-013" href="#0080-SPOT-013">0080-SPOT-013</a>).
+1. Since the order can no longer be the aggressor during continuous trading, there is no longer a requirement to hold funds to cover fees.(<a name="0080-SPOT-014" href="#0080-SPOT-014">0080-SPOT-014</a>).
```

Comment:
I think we can remove this, `0080-SPOT-013` already implies we don't need to cover fees.